README.md

```diff
@@ -10,8 +10,58 @@ The website uses the LAA Legal Adviser API. The code is hosted at https://github
 
 The REST API is https://prod.laalaa.dsd.io/.
 
+## Dependencies
+- [docker](https://www.docker.com/)
+
+## Installation for development
+
+### Docker
+
+Please ensure your internet network such as your VPN does not interfere with your build.
+
+To run container:
+
+`./run_local.sh`
+
+This shell script contains commands that build and runs the fala app.
+
+If you're not using `Docker Desktop`, you may want to have two terminals open.
+
+### Tips for developing with docker containers
+
+Code editing - You can edit the code on your local disk, with a local editor, as normal. (You don't have to edit the 
```

Comment:
I think this sentence should be this without the parenthesis:

> Code editing - You can edit the code on your local disk, with a local editor, as normal. You don't have to edit the files inside the Docker container, because your local directory is mounted into container. When you save a file, it becomes present in the container immediately, and the server restarts.